enterprise/frontend/src/embedding-sdk-shared/.storybook/preview.tsx

```diff
@@ -1,18 +1,18 @@
-import { GlobalTypes } from "@storybook/types";
+import type { GlobalTypes } from "@storybook/types";
 import { initialize, mswLoader } from "msw-storybook-addon";
 
 import { storybookThemeOptions } from "embedding-sdk-bundle/test/storybook-themes";
+import { getWindow } from "embedding-sdk-shared/lib/get-window";
+import { defineEmbeddingSdkPackageBuildInfo } from "metabase/embedding-sdk/lib/define-embedding-sdk-package-build-info";
+import { defineGlobalDependencies } from "metabase/embedding-sdk/lib/define-global-dependencies";
 
 import { availableLocales } from "./constants";
-import { defineEmbeddingSdkPackageBuildInfo } from "../frontend/src/metabase/embedding-sdk/lib/define-embedding-sdk-package-build-info";
-import { defineGlobalDependencies } from "../frontend/src/metabase/embedding-sdk/lib/define-global-dependencies";
-import { EMBEDDING_SDK_CONFIG } from "../frontend/src/metabase/embedding-sdk/config";
-
-// Enable SDK mode as we are previewing the SDK code
-EMBEDDING_SDK_CONFIG.isEmbeddingSdk = true;
 
 // To run initialization side effects like Mantine styles, dayjs plugins, etc
-import "metabase/embedding-sdk/vendors-side-effects";
+import "embedding-sdk-bundle";
+
+// To load the SDK bundle
+delete getWindow().METABASE_EMBEDDING_SDK_BUNDLE;
```

Comment:
The `import "embedding-sdk-bundle";` defines `window.METABASE_EMBEDDING_SDK_BUNDLE` that prevents the SDK bundle loading.

I can wrap it within `isStorybookActive` in the place of definition, or just remove this field here, so i go with the option #2

Reply:
We have 2 types of stories:
- that use MetabaseProvider and we have to load the SDK bundle. This line is for this case. These stories are defined in `embedding-sdk-package`
- that use commonSdkStoryWrapper and use ComponentProvider, we don't load the SDK bundle there and should not load things from the SDK package. These stories are defined on the level of the embedding-sdk-bundle

Reply:
Let's move it to decorators, and we have to fix the SdkBreadcrumbs circular references issue (not as part of this PR)

Reply:
If you mean the initial import in the `preview.tsx` files, let's probably keep it in the decorators as you suggested.
At least it is not silently fixes the SdkBreadcrumbs error, so it is still here and we have properly fix it by resolving circular references